### ISSUE TITLE
Uncaught TypeError: Cannot read property 'type' of null in assertions.js

### ISSUE THREAD

**Issue by lencioni** [rackt/react-a11y#5]
First off, great idea. I'm really excited about the work being done on this project.

Now, I just tried dropping it into my app and I quickly ran into this showstopping error:

> Uncaught TypeError: Cannot read property 'type' of null

Which seems to be happening on this line:

```javascript
if (hasText) return;else if (typeof child === "string") hasText = true;else if (child.type === "img" && child.props.alt) hasText = true;else if (child.props.children) hasText = hasChildTextNode(child.props, child.props.children);
```

Specifically, it seems that `child` is `null` in this case, so when you try to do `child.type`, we get an explosion. Adding a check like:

```javascript
else if (child === null) return;
```

before the `child.type` check prevents this from happening, but I'm not really sure if that's the correct solution. Let me know if it is, or if there is a better solution, and I'll be happy to submit a PR.

**Comment by nelix** [comment]
I put up a pr for this #6

**Comment by limscoder** [comment]
I also put up a pr for this with some tests. undefined also breaks.

**Comment by nelix** [comment]
@limscoder oh, nice.

**Comment by lencioni** [comment]
I think this was closed by 51bbbaafaaa44dc0e950fb32f05e4e2c1a8d2cb0 and has been released.
